include/zephyr/net/ethernet.h

```diff
@@ -913,6 +934,29 @@ enum ethernet_hw_caps net_eth_get_hw_capabilities(struct net_if *iface)
 	return eth->get_capabilities(net_if_get_device(iface));
 }
 
+/**
+ * @brief Return ethernet device hardware configuration information.
+ *
+ * @param iface Network interface
+ * @param config Ethernet configuration
+ *
+ * @return 0 if ok, <0 if error
+ */
+static inline
+enum ethernet_hw_caps net_eth_get_hw_config(struct net_if *iface, enum ethernet_config_type type,
+struct ethernet_config *config)
```

Comment:
nit: Please align with the opening bracket.

Reply:
Done. Btw., the return type was also incorrect, this is fixed as well.